dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -154,6 +154,10 @@ public T findPublishedById(long id) {
         return list(namedQuery("io.dockstore.webservice.core." + typeOfT.getSimpleName() + ".getEntryLiteByUserId").setParameter("userId", userId));
     }
 
+    public List<Entry> findMyEntries(long userId) {
```

Comment:
I think this should return a `List<T>` intstead of `List<Entry>`. Then you shouldn't need to do the casts in UserResource.

Reply:
complained apparently for services, so I'll just leave that one. Also found a few more places that could swap out to use the namedQueries

Reply:
Complained for bioWorkflow too...

Reply:
How did it complain? I remember talking about this with you, but at that time didn't realize the it was in this class with generics.

Reply:
just that Service and BioWorkflow weren't of type Workflow, which is weird.